develop-guide/writeparser/writeparser.rst

```diff
@@ -827,24 +846,23 @@ Using ``parsergen`` to create a parser class is particularly useful when you don
 
 Now that you've finished writing your parser, the time has come to test it!
 
-There are currently two mechanisms to create tests as the testing strategy is
-in a state of transition. The reasoning for the new testing type is to avoid
-merge conflicts, removing duplicate boilerplate code, ensuring tests are built,
-and making the process overall easier. However, in the meantime unfortunately
-a user may create different tests depending on the OS of the parser that
-they wrote.
+There are currently two types of tests, those being Folder Based and Unittest Based.
+The testing strategy is currently in a state of transition.
+Moving to folder based testing helps us avoid merge conflicts and duplicate boilerplate code
+while ensuring tests are faster and easier create and use.
+When creating tests for a new parser do so in the Folder Based style.
 
-:ref:`Folder based testing <folder_based_testing>` is currently supported for:
+:ref:`Folder based testing <folder_based_testing>` is currently completed for:
 
   * ASA
   * IOS
   * IOSXE
 
-All other OS's use :ref:`unittest based testing <unittest_based_testing>`.
+All other OS's are in transition from :ref:`unittest based testing <unittest_based_testing>`.
 
 .. important::
     If you want to contribute your new parser to the open-source |pyATS| feature
-    libraries and components, you must attach testing results for each parser
+    libraries and components, you must attach Folder Based testing results for each parser
     that you want to contribute.
```

Comment:
```suggestion
    libraries and components, you must attach Folder Based test files for each parser
    that you want to contribute. If your parser is for an OS that is not included in the 
    above list, then submit unittest test files instead.
```

Maybe we can something like that? Just in case user is submitting something weird. Not sure if that'll happen, but it's probably good to have something like that

Reply:
What do you think?

Reply:
I asked @tahigash if we wanted to have all new tests be folder based, regardless of operating system and he said yes.

Reply:
👍